src/NumField/QQ.jl

```diff
@@ -22,6 +22,8 @@ struct ZZFracIdl <: NumFieldOrdFracIdl
   end
 end
 
+Base.hash(x::ZZIdl) = hash(gen(x))
```

Comment:
```suggestion
Base.hash(x::ZZIdl, h::UInt) = hash(gen(x), h)
```
(Laut `?hash` docstring soll man die Methode mit zwei Argumenten definieren.)